packages/@sanity/desk-tool/src/panes/documentPane/timeline/timeline.styled.tsx

```diff
@@ -0,0 +1,8 @@
+import {Box} from '@sanity/ui'
+import styled from 'styled-components'
+
+export const Root = styled(Box)`
+  overflow: auto;
+  box-sizing: border-box;
+  max-height: calc(100vh - 198px);
```

Comment:
I assume this should then also be handled by the popover re: the content in the popover in the validation list PR